src/infinite-list/index.tsx

```diff
@@ -161,8 +163,8 @@ const InfiniteList = (props: InfiniteListProps, ref: any) => {
   }, [onScrollBeginDrag, onMomentumScrollEnd, scrollViewProps, isHorizontal]);
 
   const style = useMemo(() => {
-    return {height: pageHeight};
-  }, [pageHeight]);
+    return {height: pageHeight, ...(listStyle ?? {})};
```

Comment:
Why no to return an array? 
`[{height: pageHeight}, listStyle]`

Reply:
I want to allow an override of the height property if it is needed.
And the style is an object, not an array.

Reply:
if you have 'height' in listStyle it will override whether the style is an array or an object. 
Are you sure the style in RecyclerListView does not accept an array? ViewStyle does...

Reply:
What if the listStyle is a number (from a styleSheet)? You can't spread it...